test/functionalTest/cases/3272_custom_notification_sending_0-length_request/custom_notification_sending_0-length_request.test

```diff
@@ -0,0 +1,187 @@
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Custom notification sending 0-length request
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0
+accumulatorStart --pretty-print
+
+--SHELL--
+#
+# 01. Create sub1 with payload="{ %22x%22: 1}", includePayload=false
+# 02. Create sub2 without payload, includePayload=false
+# 03. Create entity E1/T1
+# 04. Dumps acummulator (see 2 notifications, both without payload)
+#
+
+
+echo '01. Create sub1 with payload="{ %22x%22: 1}", includePayload=false'
+echo "=================================================================="
+payload='{
+  "subject": {
+    "entities": [
+      {
+        "id" : "E1"
+      }
+    ],
+    "condition": {
+      "attrs": []
+    }
+  },
+  "notification": {
+    "httpCustom": {
+      "url":      "http://localhost:'${LISTENER_PORT}'/notify",
+      "payload": "{ %22x%22: 1}",
+      "includePayload":false
+    }
+  }
+}'
+orionCurl --url /v2/subscriptions --payload "$payload"
+echo
+echo
+
+
+echo '02. Create sub2 without payload, includePayload=false'
+echo "====================================================="
+payload='{
+  "subject": {
+    "entities": [
+      {
+        "id" : "E1"
+      }
+    ],
+    "condition": {
+      "attrs": []
+    }
+  },
+  "notification": {
+    "httpCustom": {
+      "url":      "http://localhost:'${LISTENER_PORT}'/notify",
+      "includePayload":false
+    }
+  }
+}'
+orionCurl --url /v2/subscriptions --payload "$payload"
+echo
+echo
+
+
+echo "03. Create entity E1/T1"
+echo "======================="
+payload='{
+  "id": "E1",
+  "type": "T1",
+  "A1": true
+}'
+orionCurl --url /v2/entities?options=keyValues --payload "$payload"
+echo
+echo
+
+
+echo "Dumps acummulator (see 2 notifications, both without payload)"
+echo "============================================================="
+accumulatorDump
+echo
+echo
+
+
+--REGEXPECT--
+01. Create sub1 with payload="{ %22x%22: 1}", includePayload=false
+==================================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+02. Create sub2 without payload, includePayload=false
+=====================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+03. Create entity E1/T1
+=======================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/entities/E1?type=T1
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+Dumps acummulator (see 2 notifications, both without payload)
```

Comment:
Note that both notifications in the test actually have payload... so it's not working as expected. See related comment: https://github.com/telefonicaid/fiware-orion/pull/3683/files?file-filters%5B%5D=.cpp&file-filters%5B%5D=.test#r576785475